src/Components/ItemList/ItemList.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Item from '../Item/Item';
+
+
+const ItemList = ({ allItems }) => (
+	<ul>
+		{allItems.map(item => (
+			<li key = {item.value}> 
+    			<Item value = {item.value} /> 
```

Comment:
Codacy found an issue: [Mixed spaces and tabs.](https://app.codacy.com/gh/WebHeroSchool/todo_react_project/pullRequest?prid=5893380)